### PR TITLE
README: Remove outdated information about launchers

### PR DESCRIPTION
**Pull request by Entropynines**
Please format your title with what portion of the project this pull request is
targeting and what it's changing.

ex. "MyGame4: implement new game" or "Docs: add new guide for customizing MyGame3"

## What is this fixing or adding?
Removed the section mentioning Windows-only installers.

## How was this tested?
I read it

### CODE REVIEW

**Review by Ixrec — APPROVED** [review]
The rewording here LGTM, especially the change in who we're suggesting should run from source.

(I cannot personally confirm that the launchers really work on non-Windows, but we probably don't need to confirm that)

File: README.md [hunk 1]
```diff
@@ -83,9 +83,9 @@ We recognize that there is a strong community of incredibly smart people that ha
 Archipelago was directly forked from bonta0's `multiworld_31` branch of ALttPEntranceRandomizer (this project has a long legacy of its own, please check it out linked above) on January 12, 2020. The repository was then named to _MultiWorld-Utilities_ to better encompass its intended function. As Archipelago matured, then known as "Berserker's MultiWorld" by some, we found it necessary to transform our repository into a root level repository (as opposed to a 'forked repo') and change the name (which came later) to better reflect our project.
 
 ## Running Archipelago
-For most people all you need to do is head over to the [releases](https://github.com/ArchipelagoMW/Archipelago/releases) page then download and run the appropriate installer. The installers function on Windows only.
+For most people all you need to do is head over to the [releases](https://github.com/ArchipelagoMW/Archipelago/releases) page then download and run the appropriate installer.
```
Comment (neocerber) [inline README.md:86]: Suggestion: 
(...) the appropriate installer if on Windows or the AppImage on Linux.
From googl-ing, it seems that an AppImage is not an installer per-se. Also, I think macOs still need to be run from source (see https://archipelago.gg/tutorial/Archipelago/mac/en).

**Review by neocerber — COMMENTED** [review]
English is my second language and I only use AP on Linux. Also, first day in my life I do github review thingy.

Except for the suggestion made in this review, this pull request seems to be all good! It enhances the documentation by better corresponding to AP!

File: README.md [hunk 2]
```diff
@@ -83,9 +83,9 @@ We recognize that there is a strong community of incredibly smart people that ha
...
 
 ## Running Archipelago
-For most people all you need to do is head over to the [releases](https://github.com/ArchipelagoMW/Archipelago/releases) page then download and run the appropriate installer. The installers function on Windows only.
+For most people all you need to do is head over to the [releases](https://github.com/ArchipelagoMW/Archipelago/releases) page then download and run the appropriate installer. 
 
-If you are running Archipelago from a non-Windows system then the likely scenario is that you are comfortable running source code directly. Please see our doc on [running Archipelago from source](docs/running%20from%20source.md).
+If you are or are planning to become a developer, or you are running on a platform with no compiled releases available, please see our doc on [running Archipelago from source](docs/running%20from%20source.md).
```
Comment (nicholassaylor) [inline README.md:88]:
```suggestion
If you are currently a developer, planning to become one, or are running on a platform with no compiled releases available, please see our doc on [running Archipelago from source](docs/running%20from%20source.md).
```
Seems better to just make it a list here, wording for being a developer was a bit awkward, feel free to change what I suggest

**Review by nicholassaylor — APPROVED** [review]
Changes seem reasonable, no major issues, just a wording nitpick
Linux does have AppImages now so ultimately should be handled
LGTM

File: README.md (hunk 2 above)
Comment (Exempt-Medic) [inline README.md:88]: Honestly, I think "planning to become one" can be dropped entirely from this sentence. It doesn't really help, much.
```suggestion
If you are a developer or are running on a platform with no compiled releases available, please see our doc on [running Archipelago from source](docs/running%20from%20source.md).
```

Comment (Entropynines) [reply README.md]: agreed, dropped that part

**Review by Exempt-Medic — APPROVED** [review]
The wording looks good and removes the presumptuous bit of text in the second paragraph. I honestly know very little about Linux or Mac, so I can't speak to the _accuracy_ of the sentences, but I'll assume they're true and a core dev will have to look at this anyhow.

**Review by ThePhar — APPROVED** [review]

**Pull request merged**